packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -190,6 +193,42 @@ export class Meter implements api.Meter {
     return valueObserver;
   }
 
+  /**
+   * Creates a new up down sum observer metric.
+   * @param name the name of the metric.
+   * @param [options] the metric options.
+   * @param [callback] the value observer callback
+   */
+  createUpDownSumObserver(
+    name: string,
+    options: api.MetricOptions = {},
+    callback?: (
+      observerResult: api.ObserverResult
+    ) => Promise<unknown> | unknown
+  ): api.ValueObserver {
+    if (!this._isValidName(name)) {
```

Comment:
This piece of code is repeated in all the metrics, I believe we can do something better than this.

Reply:
I was already thinking about it but there are some small differences. I would wait until the last metric is implemented and then try to refactor it to simply avoid pushing too many things at once as well as avoid merging conflicts in this place, WDYT ?

Reply:
Sure makes sense to me.